novawallet/Modules/Vote/Governance/Validating/GovernanceValidatorFactory.swift

```diff
@@ -2,7 +2,7 @@ import Foundation
 import BigInt
 import SoraFoundation
 
-protocol GovernanceValidatorFactoryProtocol: BaseDataValidatingFactoryProtocol {
```

Comment:
I wouldn't mixed up delegations in governance and proxies: they are different notions. Let's keep them separate.